lib/cli/index.js

```diff
@@ -2873,9 +2873,15 @@ export async function createNodejsBom(path, options) {
       }
       const rdeplist = [];
       if (parsedList.dependenciesList && parsedList.dependenciesList) {
+        // copyright (c) 2025 Atlassian US, Inc.
+        // First read package.json to get direct dependencies
+        const pkgData = JSON.parse(readFileSync(packageJsonF, "utf8"));
```

Comment:
Should we move this logic to outside the for loop? Also how would this work in monorepos with multiple package.json and yarn.lock files?